acme_diags/plot/cartopy/streamflow_plot.py

```diff
@@ -1,24 +1,25 @@
 from __future__ import print_function
 
-import matplotlib
-import matplotlib.lines as lines
-import numpy as np
 import os
 
-matplotlib.use('Agg')
-import matplotlib.pyplot as plt
-import matplotlib.colors as colors
-import numpy
-import scipy.stats
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
-from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
 import cdutil
+import matplotlib
+import matplotlib.colors as colors
+import matplotlib.lines as lines
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.stats
+from cartopy.mpl.ticker import LatitudeFormatter, LongitudeFormatter
+
 from acme_diags.derivations.default_regions import regions_specs
 from acme_diags.driver.utils.general import get_output_dir
 
-plotTitle = {'fontsize': 11.5}
-plotSideTitle = {'fontsize': 9.5}
+matplotlib.use("Agg")
```

Comment:
This is another example of  a line of code that needs to go before imports. I think I missed it in a few other `plot` files.